Gemfile

```diff
@@ -86,14 +85,14 @@ gem "rack-throttle"
 gem "secure_headers", "~> 2.5.0"
 
 # Frontend
-gem "backbone-on-rails", "~>0.9.10.0" # Legacy js
+gem "jquery-rails"
 gem "bootstrap", "~> 4.0.0.alpha3" # Bootstrap 4 - used for revised stylesheets
 gem "chartkick" # Display charts
 gem "coffee-rails"
 gem "groupdate" # Required for charts
-gem "jquery-datatables-rails", "~>3.4.0"
```

Comment:
Can be removed, doesn't need to be a gem, might not really need to be included anymore (only used by `/manufacturers`?)